### DISCUSSION TITLE
2024-06-18(화)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#166] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by devwqc** [comment]
### 완료한 일
- 레이아웃 작업
- 소개 화면 작업
- 404 페이지 작업
- 로딩 페이지 작업

### 해야할 일
- 홈, 검색, 찜 페이지

### 어려운 일
- 파이팅!!👍💯

**Comment by Seoin02** [comment]
### 완료한 일
- 회원가입 로직 수정&useAuth 수정
- 회원정보 수정(전화번호)구현
- 회원 탈퇴 구현
- next 미들웨어 세팅

### 해야할 일
- 온보딩 페이지 / put API 요청 / 그래픽 작업
- 프로필 수정 API 연결
- signup 페이지 닉네임 중복 검사 구현

### 어려운 일
- 반려동물 선택 웨 안뒈지,,,

**Comment by q45402sk** [comment]
### 완료한 일
카드 컴포넌트 수정 
deliveryEditPage 수정
myDeliveryPage (pr 올릴 예정)

### 해야할 일
purchaseDeliveryPage
상품상세 스크롤 적용

### 어려운 일

**Comment by seolsis** [comment]
### 완료한 일
- 결제페이지 데이터 받아오기
### 해야할 일
- recomended 컴포넌트 추가
- paymentKey 이해
- 결제 취소 페이지
- toast
### 어려운 일

**Comment by minyoung0503** [comment]
### 완료한 일

### 해야할 일
my -> reviewPage API
상품 옵션 bottom sheet

### 어려운 일
